### PR TITLE
Feature/e UI 8483

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8483

### Change description ###
Add full name input

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by Daniel-Lam — APPROVED** [review]
@MateiHMCTS - approved

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-write/query-write-respond-to-query/query-write-respond-to-query.component.html [hunk 1]
```diff
@@ -22,6 +22,25 @@ <h1 class="govuk-heading-l govuk-!-margin-bottom-0">{{ 'Query details' | rpxTran
...
+          <label class="govuk-label govuk-label--m govuk-!-font-weight-bold" for="query-respond-fullName">
+            {{ 'Full name' | rpxTranslate }}
+          </label>
+        </div>
+        <div id="query-respond-fullName-hint" class="govuk-hint">
+          {{ 'Your full name must be included here' | rpxTranslate }}
+        </div>
+        <p id="query-respond-fullName-error" class="govuk-error-message"
+           *ngIf="formGroup.get('fullName')?.errors?.length">
+          <span class="govuk-visually-hidden">{{ 'Error:' | rpxTranslate }}</span> {{ 'Enter a message' | rpxTranslate }}
```
Comment (DavidJayakumar) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-write/query-write-respond-to-query/query-write-respond-to-query.component.html:37]: @MateiHMCTS Error message should be 'Enter your fullname'.

**Review by DavidJayakumar — CHANGES_REQUESTED** [review]
Error message is incorrect

**Review by DavidJayakumar — APPROVED** [review]
LGTM

**Review by Daniel-Lam — APPROVED** [review]
@MateiHMCTS - re-approved

**Review by DavidJayakumar — DISMISSED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by Daniel-Lam — APPROVED** [review]
@MateiHMCTS - approved

**Review by DavidJayakumar — DISMISSED** [review]
Looks good

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Pull request merged**